tor/core/posts.py

```diff
@@ -160,6 +163,7 @@ def get_blossom_submission(submission: Submission, cfg: Config) -> Dict:
     response = cfg.blossom.get_submission(url=submission.url)
     if response.status == BlossomStatus.ok:
         return response.data[0]
-    else:
-        # If we are here, this means that the current submission is not yet in Blossom.
-        return create_blossom_submission(submission, cfg)
+
+    # TODO: Submit it to fix the data. It was posted and someone claimed,
+    # so it must be an error in our bookkeeping.
+    raise LookupError(f'No such submission for {submission.id}')
```

Comment:
This will explode on any submission created pre-Blossom -- while there aren't that many accessible anymore, it is still possible someone might find one and try to complete it.